controllers/users.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectUsers } = require("../models/users.model");
+
+exports.getUsers = (req, res) => {
+  selectUsers().then((output) => {
```

Comment:
I'd maybe just name `output` to be a bit of a better variable name for its content